packages/components/progress/src/Progress.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <component :is="tag" v-bind="$props" class="ix-progress">
```

Comment:
```suggestion
  <component :is="tag" v-bind="$props" : percent="percent" : successPercent="successPercent"  class="ix-progress">
```
可以在父组件把参数都格式化好，然后在子组件内部使用的时候，就不用在考虑数据一致性了。
相应的 convertPercent 都可以干掉。

如果还有其他需要处理的参数也可以放在父组件。